app/questionnaire/navigation.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, schema, answer_store, metadata, completed_blocks, routing_pat
         self.completed_blocks = completed_blocks
         self.routing_path = routing_path
 
+        self.completeness = Completeness(
```

Comment:
Why not use your new get_completeness() method?
I think its off that we have a method that constructs this for us yet we don't consistently use it?
To quote 'O-Town' I think we should have it "All or nothing at all"

Reply:
Ha - yep good point. I missed this when putting in that function.